enforcer/src/main/java/org/wso2/choreo/connect/enforcer/server/AuthServer.java

```diff
@@ -143,6 +144,7 @@ private static Server initServer() throws SSLException {
                 .workerEventLoopGroup(workerGroup)
                 .addService(ServerInterceptors.intercept(new ExtAuthService(), new AccessLogInterceptor()))
                 .addService(new HealthService())
+                .addService(new WebSocketFrameService())
```

Comment:
Shall we add the access log interceptor and also check whether it works

Reply:
fixed